## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 514242`
Commit: [`b3788e4c81e0bee12ce67b82b9090ede569dfd06`](https://github.com/NixOS/nixpkgs/commit/b3788e4c81e0bee12ce67b82b9090ede569dfd06) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/b3788e4c81e0bee12ce67b82b9090ede569dfd06..pull/514242/head))
Merge: [`c7e3c14c8e2aeea2e9b81abe3daca2a15b6b93f2`](https://github.com/NixOS/nixpkgs/commit/c7e3c14c8e2aeea2e9b81abe3daca2a15b6b93f2)

Logs: https://github.com/Hythera/nixpkgs-review-gha/actions/runs/25023881273


---
### `x86_64-linux`
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-git-rebase</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-git-rebase.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-git-rebase</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-git-rebase.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 85 packages built:</summary>
  <ul>
    <li>diffsitter</li>
    <li>evil-helix</li>
    <li>helix</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-blueprint</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-blueprint.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-caddyfile</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-caddyfile.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-cel</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-cel.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-dart</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-dart.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-fga</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-fga.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-groovy</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-groovy.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ledger</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ledger.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-llvm</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-llvm.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-mail</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-mail.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-mojo</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-mojo.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-nu</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-nu.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-razor</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-razor.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-rego</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-rego.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-slint</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-slint.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-typespec</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-typespec.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-vento</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-vento.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-blueprint</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-blueprint.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-caddyfile</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-caddyfile.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-cel</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-cel.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-dart</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-dart.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-fga</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-fga.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-groovy</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-groovy.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-ledger</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-ledger.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-llvm</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-llvm.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-mail</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-mail.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-mojo</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-mojo.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-nu</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-nu.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-razor</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-razor.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-rego</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-rego.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-slint</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-slint.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-typespec</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-typespec.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-vento</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-vento.dist</li>
    <li>steelix</li>
    <li>tree-sitter-grammars.tree-sitter-blueprint</li>
    <li>tree-sitter-grammars.tree-sitter-caddyfile</li>
    <li>tree-sitter-grammars.tree-sitter-cel</li>
    <li>tree-sitter-grammars.tree-sitter-dart</li>
    <li>tree-sitter-grammars.tree-sitter-fga</li>
    <li>tree-sitter-grammars.tree-sitter-git-rebase</li>
    <li>tree-sitter-grammars.tree-sitter-groovy</li>
    <li>tree-sitter-grammars.tree-sitter-ledger</li>
    <li>tree-sitter-grammars.tree-sitter-llvm</li>
    <li>tree-sitter-grammars.tree-sitter-mail</li>
    <li>tree-sitter-grammars.tree-sitter-mojo</li>
    <li>tree-sitter-grammars.tree-sitter-nu</li>
    <li>tree-sitter-grammars.tree-sitter-razor</li>
    <li>tree-sitter-grammars.tree-sitter-rego</li>
    <li>tree-sitter-grammars.tree-sitter-slint</li>
    <li>tree-sitter-grammars.tree-sitter-typespec</li>
    <li>tree-sitter-grammars.tree-sitter-vento</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python313Packages.tree-sitter-grammars.tree-sitter-git-rebase</summary>
<pre>    
        # Massage each grammar&#x27;s path into an absolute path so later code
        # can find queries, etc.
&gt;       conf = TreeSitterConfig.model_validate_json(json_path.read_text())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E       pydantic_core._pydantic_core.ValidationError: 1 validation error for TreeSitterConfig
E       metadata.links.funding
E         Input should be a valid URL, input is empty [type=url_parsing, input_value=&#x27;&#x27;, input_type=str]
E           For further information visit https://errors.pydantic.dev/2.12/v/url_parsing

/nix/store/q76m4s711azwvy7n7m28d7yghzh9yc9a-python3.13-python-tree-sitter-git-rebase-1.0.0/lib/python3.13/site-packages/tree_sitter_git_rebase/__init__.py:24: ValidationError
=============================== warnings summary ===============================
tests/test_language.py::test_language
  /build/python-tree-sitter-git-rebase-source/tests/test_language.py:11: DeprecationWarning: int argument support is deprecated
    lang = Language(language())

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_language.py::test_config - pydantic_core._pydantic_core.ValidationError: 1 validation error for TreeSi...
==================== 1 failed, 1 passed, 1 warning in 0.73s ====================</pre>
</details>
<details>
<summary>python314Packages.tree-sitter-grammars.tree-sitter-git-rebase</summary>
<pre>    
        # Massage each grammar&#x27;s path into an absolute path so later code
        # can find queries, etc.
&gt;       conf = TreeSitterConfig.model_validate_json(json_path.read_text())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E       pydantic_core._pydantic_core.ValidationError: 1 validation error for TreeSitterConfig
E       metadata.links.funding
E         Input should be a valid URL, input is empty [type=url_parsing, input_value=&#x27;&#x27;, input_type=str]
E           For further information visit https://errors.pydantic.dev/2.12/v/url_parsing

/nix/store/571l3cr3n08kdhb1vh66wyjx7ccin9nk-python3.14-python-tree-sitter-git-rebase-1.0.0/lib/python3.14/site-packages/tree_sitter_git_rebase/__init__.py:24: ValidationError
=============================== warnings summary ===============================
tests/test_language.py::test_language
  /build/python-tree-sitter-git-rebase-source/tests/test_language.py:11: DeprecationWarning: int argument support is deprecated
    lang = Language(language())

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_language.py::test_config - pydantic_core._pydantic_core.ValidationError: 1 validation error for TreeSi...
==================== 1 failed, 1 passed, 1 warning in 0.34s ====================</pre>
</details>
</details>

---
### `aarch64-linux`
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-git-rebase</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-git-rebase.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-git-rebase</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-git-rebase.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 85 packages built:</summary>
  <ul>
    <li>diffsitter</li>
    <li>evil-helix</li>
    <li>helix</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-blueprint</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-blueprint.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-caddyfile</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-caddyfile.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-cel</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-cel.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-dart</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-dart.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-fga</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-fga.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-groovy</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-groovy.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ledger</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ledger.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-llvm</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-llvm.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-mail</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-mail.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-mojo</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-mojo.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-nu</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-nu.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-razor</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-razor.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-rego</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-rego.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-slint</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-slint.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-typespec</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-typespec.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-vento</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-vento.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-blueprint</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-blueprint.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-caddyfile</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-caddyfile.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-cel</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-cel.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-dart</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-dart.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-fga</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-fga.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-groovy</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-groovy.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-ledger</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-ledger.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-llvm</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-llvm.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-mail</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-mail.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-mojo</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-mojo.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-nu</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-nu.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-razor</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-razor.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-rego</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-rego.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-slint</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-slint.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-typespec</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-typespec.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-vento</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-vento.dist</li>
    <li>steelix</li>
    <li>tree-sitter-grammars.tree-sitter-blueprint</li>
    <li>tree-sitter-grammars.tree-sitter-caddyfile</li>
    <li>tree-sitter-grammars.tree-sitter-cel</li>
    <li>tree-sitter-grammars.tree-sitter-dart</li>
    <li>tree-sitter-grammars.tree-sitter-fga</li>
    <li>tree-sitter-grammars.tree-sitter-git-rebase</li>
    <li>tree-sitter-grammars.tree-sitter-groovy</li>
    <li>tree-sitter-grammars.tree-sitter-ledger</li>
    <li>tree-sitter-grammars.tree-sitter-llvm</li>
    <li>tree-sitter-grammars.tree-sitter-mail</li>
    <li>tree-sitter-grammars.tree-sitter-mojo</li>
    <li>tree-sitter-grammars.tree-sitter-nu</li>
    <li>tree-sitter-grammars.tree-sitter-razor</li>
    <li>tree-sitter-grammars.tree-sitter-rego</li>
    <li>tree-sitter-grammars.tree-sitter-slint</li>
    <li>tree-sitter-grammars.tree-sitter-typespec</li>
    <li>tree-sitter-grammars.tree-sitter-vento</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>python313Packages.tree-sitter-grammars.tree-sitter-git-rebase</summary>
<pre>    
        # Massage each grammar&#x27;s path into an absolute path so later code
        # can find queries, etc.
&gt;       conf = TreeSitterConfig.model_validate_json(json_path.read_text())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E       pydantic_core._pydantic_core.ValidationError: 1 validation error for TreeSitterConfig
E       metadata.links.funding
E         Input should be a valid URL, input is empty [type=url_parsing, input_value=&#x27;&#x27;, input_type=str]
E           For further information visit https://errors.pydantic.dev/2.12/v/url_parsing

/nix/store/07zfqiy9s327h81zfx9cs4s7sb0c010q-python3.13-python-tree-sitter-git-rebase-1.0.0/lib/python3.13/site-packages/tree_sitter_git_rebase/__init__.py:24: ValidationError
=============================== warnings summary ===============================
tests/test_language.py::test_language
  /build/python-tree-sitter-git-rebase-source/tests/test_language.py:11: DeprecationWarning: int argument support is deprecated
    lang = Language(language())

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_language.py::test_config - pydantic_core._pydantic_core.ValidationError: 1 validation error for TreeSi...
==================== 1 failed, 1 passed, 1 warning in 0.23s ====================</pre>
</details>
<details>
<summary>python314Packages.tree-sitter-grammars.tree-sitter-git-rebase</summary>
<pre>    
        # Massage each grammar&#x27;s path into an absolute path so later code
        # can find queries, etc.
&gt;       conf = TreeSitterConfig.model_validate_json(json_path.read_text())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E       pydantic_core._pydantic_core.ValidationError: 1 validation error for TreeSitterConfig
E       metadata.links.funding
E         Input should be a valid URL, input is empty [type=url_parsing, input_value=&#x27;&#x27;, input_type=str]
E           For further information visit https://errors.pydantic.dev/2.12/v/url_parsing

/nix/store/96knaj7i5r88l7h8lw43zhlzqbrya525-python3.14-python-tree-sitter-git-rebase-1.0.0/lib/python3.14/site-packages/tree_sitter_git_rebase/__init__.py:24: ValidationError
=============================== warnings summary ===============================
tests/test_language.py::test_language
  /build/python-tree-sitter-git-rebase-source/tests/test_language.py:11: DeprecationWarning: int argument support is deprecated
    lang = Language(language())

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_language.py::test_config - pydantic_core._pydantic_core.ValidationError: 1 validation error for TreeSi...
==================== 1 failed, 1 passed, 1 warning in 0.24s ====================</pre>
</details>
</details>

---
### `x86_64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-git-rebase</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-git-rebase.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-git-rebase</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-git-rebase.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 85 packages built:</summary>
  <ul>
    <li>diffsitter</li>
    <li>evil-helix</li>
    <li>helix</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-blueprint</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-blueprint.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-caddyfile</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-caddyfile.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-cel</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-cel.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-dart</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-dart.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-fga</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-fga.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-groovy</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-groovy.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ledger</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ledger.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-llvm</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-llvm.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-mail</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-mail.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-mojo</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-mojo.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-nu</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-nu.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-razor</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-razor.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-rego</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-rego.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-slint</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-slint.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-typespec</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-typespec.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-vento</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-vento.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-blueprint</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-blueprint.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-caddyfile</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-caddyfile.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-cel</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-cel.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-dart</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-dart.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-fga</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-fga.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-groovy</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-groovy.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-ledger</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-ledger.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-llvm</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-llvm.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-mail</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-mail.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-mojo</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-mojo.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-nu</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-nu.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-razor</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-razor.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-rego</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-rego.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-slint</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-slint.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-typespec</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-typespec.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-vento</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-vento.dist</li>
    <li>steelix</li>
    <li>tree-sitter-grammars.tree-sitter-blueprint</li>
    <li>tree-sitter-grammars.tree-sitter-caddyfile</li>
    <li>tree-sitter-grammars.tree-sitter-cel</li>
    <li>tree-sitter-grammars.tree-sitter-dart</li>
    <li>tree-sitter-grammars.tree-sitter-fga</li>
    <li>tree-sitter-grammars.tree-sitter-git-rebase</li>
    <li>tree-sitter-grammars.tree-sitter-groovy</li>
    <li>tree-sitter-grammars.tree-sitter-ledger</li>
    <li>tree-sitter-grammars.tree-sitter-llvm</li>
    <li>tree-sitter-grammars.tree-sitter-mail</li>
    <li>tree-sitter-grammars.tree-sitter-mojo</li>
    <li>tree-sitter-grammars.tree-sitter-nu</li>
    <li>tree-sitter-grammars.tree-sitter-razor</li>
    <li>tree-sitter-grammars.tree-sitter-rego</li>
    <li>tree-sitter-grammars.tree-sitter-slint</li>
    <li>tree-sitter-grammars.tree-sitter-typespec</li>
    <li>tree-sitter-grammars.tree-sitter-vento</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>python313Packages.tree-sitter-grammars.tree-sitter-git-rebase</summary>
<pre>    
        # Massage each grammar&#x27;s path into an absolute path so later code
        # can find queries, etc.
&gt;       conf = TreeSitterConfig.model_validate_json(json_path.read_text())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E       pydantic_core._pydantic_core.ValidationError: 1 validation error for TreeSitterConfig
E       metadata.links.funding
E         Input should be a valid URL, input is empty [type=url_parsing, input_value=&#x27;&#x27;, input_type=str]
E           For further information visit https://errors.pydantic.dev/2.12/v/url_parsing

/nix/store/09cdkdw12w6w0f9aww5adzzxly3r8cws-python3.13-python-tree-sitter-git-rebase-1.0.0/lib/python3.13/site-packages/tree_sitter_git_rebase/__init__.py:24: ValidationError
=============================== warnings summary ===============================
tests/test_language.py::test_language
  /nix/build/nix-7676-4213235176/python-tree-sitter-git-rebase-source/tests/test_language.py:11: DeprecationWarning: int argument support is deprecated
    lang = Language(language())

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_language.py::test_config - pydantic_core._pydantic_core.ValidationError: 1 validation error for TreeSi...
==================== 1 failed, 1 passed, 1 warning in 0.34s ====================</pre>
</details>
<details>
<summary>python314Packages.tree-sitter-grammars.tree-sitter-git-rebase</summary>
<pre>    
        # Massage each grammar&#x27;s path into an absolute path so later code
        # can find queries, etc.
&gt;       conf = TreeSitterConfig.model_validate_json(json_path.read_text())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E       pydantic_core._pydantic_core.ValidationError: 1 validation error for TreeSitterConfig
E       metadata.links.funding
E         Input should be a valid URL, input is empty [type=url_parsing, input_value=&#x27;&#x27;, input_type=str]
E           For further information visit https://errors.pydantic.dev/2.12/v/url_parsing

/nix/store/liwx89wcslxr3x3vnf2g2fi7lhcs1wcg-python3.14-python-tree-sitter-git-rebase-1.0.0/lib/python3.14/site-packages/tree_sitter_git_rebase/__init__.py:24: ValidationError
=============================== warnings summary ===============================
tests/test_language.py::test_language
  /nix/build/nix-7676-4213235177/python-tree-sitter-git-rebase-source/tests/test_language.py:11: DeprecationWarning: int argument support is deprecated
    lang = Language(language())

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_language.py::test_config - pydantic_core._pydantic_core.ValidationError: 1 validation error for TreeSi...
==================== 1 failed, 1 passed, 1 warning in 0.39s ====================</pre>
</details>
</details>

---
### `aarch64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 4 packages failed to build:</summary>
  <ul>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-git-rebase</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-git-rebase.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-git-rebase</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-git-rebase.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 85 packages built:</summary>
  <ul>
    <li>diffsitter</li>
    <li>evil-helix</li>
    <li>helix</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-blueprint</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-blueprint.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-caddyfile</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-caddyfile.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-cel</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-cel.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-dart</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-dart.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-fga</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-fga.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-groovy</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-groovy.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ledger</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ledger.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-llvm</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-llvm.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-mail</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-mail.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-mojo</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-mojo.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-nu</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-nu.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-razor</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-razor.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-rego</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-rego.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-slint</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-slint.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-typespec</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-typespec.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-vento</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-vento.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-blueprint</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-blueprint.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-caddyfile</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-caddyfile.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-cel</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-cel.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-dart</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-dart.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-fga</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-fga.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-groovy</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-groovy.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-ledger</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-ledger.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-llvm</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-llvm.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-mail</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-mail.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-mojo</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-mojo.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-nu</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-nu.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-razor</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-razor.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-rego</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-rego.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-slint</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-slint.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-typespec</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-typespec.dist</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-vento</li>
    <li>python314Packages.tree-sitter-grammars.tree-sitter-vento.dist</li>
    <li>steelix</li>
    <li>tree-sitter-grammars.tree-sitter-blueprint</li>
    <li>tree-sitter-grammars.tree-sitter-caddyfile</li>
    <li>tree-sitter-grammars.tree-sitter-cel</li>
    <li>tree-sitter-grammars.tree-sitter-dart</li>
    <li>tree-sitter-grammars.tree-sitter-fga</li>
    <li>tree-sitter-grammars.tree-sitter-git-rebase</li>
    <li>tree-sitter-grammars.tree-sitter-groovy</li>
    <li>tree-sitter-grammars.tree-sitter-ledger</li>
    <li>tree-sitter-grammars.tree-sitter-llvm</li>
    <li>tree-sitter-grammars.tree-sitter-mail</li>
    <li>tree-sitter-grammars.tree-sitter-mojo</li>
    <li>tree-sitter-grammars.tree-sitter-nu</li>
    <li>tree-sitter-grammars.tree-sitter-razor</li>
    <li>tree-sitter-grammars.tree-sitter-rego</li>
    <li>tree-sitter-grammars.tree-sitter-slint</li>
    <li>tree-sitter-grammars.tree-sitter-typespec</li>
    <li>tree-sitter-grammars.tree-sitter-vento</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python313Packages.tree-sitter-grammars.tree-sitter-git-rebase</summary>
<pre>    
        # Massage each grammar&#x27;s path into an absolute path so later code
        # can find queries, etc.
&gt;       conf = TreeSitterConfig.model_validate_json(json_path.read_text())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E       pydantic_core._pydantic_core.ValidationError: 1 validation error for TreeSitterConfig
E       metadata.links.funding
E         Input should be a valid URL, input is empty [type=url_parsing, input_value=&#x27;&#x27;, input_type=str]
E           For further information visit https://errors.pydantic.dev/2.12/v/url_parsing

/nix/store/rwbgb511njbky1qwqwfdx45rmw4c8b8r-python3.13-python-tree-sitter-git-rebase-1.0.0/lib/python3.13/site-packages/tree_sitter_git_rebase/__init__.py:24: ValidationError
=============================== warnings summary ===============================
tests/test_language.py::test_language
  /nix/build/nix-6193-1566086815/python-tree-sitter-git-rebase-source/tests/test_language.py:11: DeprecationWarning: int argument support is deprecated
    lang = Language(language())

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_language.py::test_config - pydantic_core._pydantic_core.ValidationError: 1 validation error for TreeSi...
==================== 1 failed, 1 passed, 1 warning in 0.30s ====================</pre>
</details>
<details>
<summary>python314Packages.tree-sitter-grammars.tree-sitter-git-rebase</summary>
<pre>    
        # Massage each grammar&#x27;s path into an absolute path so later code
        # can find queries, etc.
&gt;       conf = TreeSitterConfig.model_validate_json(json_path.read_text())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E       pydantic_core._pydantic_core.ValidationError: 1 validation error for TreeSitterConfig
E       metadata.links.funding
E         Input should be a valid URL, input is empty [type=url_parsing, input_value=&#x27;&#x27;, input_type=str]
E           For further information visit https://errors.pydantic.dev/2.12/v/url_parsing

/nix/store/9i8n582p5lbn0icgplqqqxpr74hzmmhi-python3.14-python-tree-sitter-git-rebase-1.0.0/lib/python3.14/site-packages/tree_sitter_git_rebase/__init__.py:24: ValidationError
=============================== warnings summary ===============================
tests/test_language.py::test_language
  /nix/build/nix-6193-1566086819/python-tree-sitter-git-rebase-source/tests/test_language.py:11: DeprecationWarning: int argument support is deprecated
    lang = Language(language())

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_language.py::test_config - pydantic_core._pydantic_core.ValidationError: 1 validation error for TreeSi...
==================== 1 failed, 1 passed, 1 warning in 0.33s ====================</pre>
</details>
</details>

--- a/pkgs/by-name/tr/tree-sitter/grammars/grammar-sources.nix
+++ b/pkgs/by-name/tr/tree-sitter/grammars/grammar-sources.nix
@@ -813,10 +813,14 @@
   };
 
   git-rebase = {
-    version = "0-unstable-2024-07-22";
+    version = "1.0.0";
     url = "github:the-mikedavis/tree-sitter-git-rebase";
-    rev = "bff4b66b44b020d918d67e2828eada1974a966aa";
-    hash = "sha256-k4C7dJUkvQxIxcaoVmG2cBs/CeYzVqrip2+2mRvHtZc=";
+    hash = "sha256-EV/Ecfzu3jZ5BQynRxn8NJ+lfi9i5ixs+u1e72uZBJA=";
+    postPatch = ''
+      # The funding url is empty, which will result in failing tests for the python package
+      # tree-sitter-grammars.tree-sitter.git-rebase.
+      jq 'del(.metadata.links.funding)' tree-sitter.json > tree-sitter.json.tmp && mv tree-sitter.json.tmp tree-sitter.json
+    '';
     meta = {
       license = lib.licenses.mit;
       maintainers = with lib.maintainers; [

--- a/pkgs/by-name/tr/tree-sitter/grammars/grammar-sources.nix
+++ b/pkgs/by-name/tr/tree-sitter/grammars/grammar-sources.nix
@@ -1472,10 +1472,10 @@
   };
 
   llvm = {
-    version = "0-unstable-2024-10-07";
+    version = "0-unstable-2025-08-22";
     url = "github:benwilliamgraham/tree-sitter-llvm";
-    rev = "c14cb839003348692158b845db9edda201374548";
-    hash = "sha256-L3XwPhvwIR/mUbugMbaHS9dXyhO7bApv/gdlxQ+2Bbo=";
+    rev = "2914786ae6774d4c4e25a230f4afe16aa68fe1c1";
+    hash = "sha256-jBSotMFsBUcgQrWH5p8EiywG00+v9QqePcUTI6ZqAkw=";
     meta = {
       license = lib.licenses.mit;
     };

--- a/pkgs/by-name/tr/tree-sitter/grammars/grammar-sources.nix
+++ b/pkgs/by-name/tr/tree-sitter/grammars/grammar-sources.nix
@@ -1462,10 +1462,10 @@
   };
 
   ledger = {
-    version = "0-unstable-2025-05-04";
+    version = "0-unstable-2026-03-20";
     url = "github:cbarrete/tree-sitter-ledger";
-    rev = "96c92d4908a836bf8f661166721c98439f8afb80";
-    hash = "sha256-L2xUTItnQ/bcieasItrozjAEJLm/fsUUyMex2juCnjw=";
+    rev = "22a1ab8195c1f6e808679f803007756fe7638c6f";
+    hash = "sha256-62xgcEavI5RKi77sbEnx9f3hA4faFeUCw0/uec8Nx3k=";
     meta = {
       license = lib.licenses.mit;
     };

